br/tests/lightning_bom_file/data/mytest.testtbl.csv

```diff
@@ -0,0 +1,6 @@
+﻿id,val1
```

Comment:
Maybe check the BOM really exists in run.sh 😂

Reply:
I remember the BOM header cannot be presented properly in the text editor...Maybe it's just ignored similarly by GitHub

Reply:
it has no visual output, but we can use `xxd` or something tools to check it. I'm afraid that the BOM header will be deleted accidentally in future.

Reply:
addressed in 97497035d8e33f955affbabfd2fb69f6d1621263